tests/services/test_service.py

```diff
@@ -136,3 +136,22 @@ def test_missing_or_invalid_type(lang_data2, service, identity):
         lang_data2['props']['newkey'] = v
         pytest.raises(
             ValidationError, service.create, identity, lang_data2)
+
+
+def test_read_all_no_cache(lang_type, lic_type, lang_data_many, service, identity):
+    """ read_all method should return all languages created in this scope. """
+
+    items = service.read_all(identity, fields=[], type='languages', cache=False)
+    assert set(lang_data_many).issubset(set([i.id for i in items._results])) 
+    #assert len(items._results) == len(lang_data_many) #TODO: Fix it to actual number with scope='module'
```

Comment:
wouldn't it better to keep it scope to this function... no need to check the length as long as the known existing ones are there.

what about actually check that the vocabulary *type* (i.e. languages) is correct in all of them, and no other vocabulary type got in the results.